test/parallel/test-require-symlink.js

```diff
@@ -56,4 +57,13 @@ function test() {
     assert(!code);
     assert(!signal);
   });
+
+  // Also verify that symlinks works for setting preserve via env variables
+  const childEnv = spawn(node, [linkScript], {
+    env: util._extend(process.env, {NODE_PRESERVE_SYMLINKS: '1'})
+  });
+  childEnv.on('close', function(code, signal) {
+    assert(!code);
```

Comment:
I noticed that as well but it's locally consistent, the surrounding code uses plain `assert()` too.